### ISSUE TITLE
Label Contrast Issue on Combination Charts

### ISSUE THREAD

**Issue by wvanrensburg** [highcharts/highcharts#23964]
#### Expected behaviour
When a spline chart's data label is over a column chart, the label color should automatically adjust its contrast to be visible

#### Actual behaviour
spline charts label color is not visible using dark or light column colors.

#### Live demo with steps to reproduce
https://jsfiddle.net/hs8oyx1j/

#### Product version
Highcharts Core,  Combination Chart of Spline and Column

#### Affected browser(s)
All

**Comment by magdalena-gut** [comment]
Hi @wvanrensburg, thanks for your feedback! Since the spline data labels may partially overlap both the columns and the background, handling this correctly is not straightforward from an implementation point of view. We’ll bring this up for internal discussion.

For now, the most universal option would be to set a background for the data labels (https://jsfiddle.net/mt42935e/
) or keep the outline.

**Comment by TorsteinHonsi** [comment]
We looked into a possible general solution here, but landed on closing this as won't fix. The thing is that the original demo explicitly set `textOutline` to `none`. The reason why it is enabled by default is precisely this case - we don't always know the background. If we keep the defaults, it looks okay: https://jsfiddle.net/highcharts/ecwyo63n/

The fix we tried was to look for a column in the background. But that is very easy to throw off, for example with a narrower column. In reality the data label must be readable against contrasted background.

For those who don’t like the default textOutline, Highcharts v13 comes with a `backgroundColor: 'contrast'` feature that is easier on the eye: https://jsfiddle.net/highcharts/ecwyo63n/3/
